ui/.env.development

```diff
@@ -1,2 +1,3 @@
 NODE_ENV="development"
-VUE_APP_API_BASE_URL="http://localhost:8000/api/"
\ No newline at end of file
+VUE_APP_API_BASE_URL="http://localhost:8000/api/"
+VUE_APP_IFRAME_BASE_URL="http://localhost:8000/"
```

Comment:
Rather than using the same url for two things, its better to delete api/ from api base url, then add /api/ in router/index.js / all files that called this var :"   Im sure its not called in many files

Reply:
I have checked, and in relation to this var, its only called in api/config. The one in cypress is hardcoded.